pkg/nimble/README.ipv6-over-ble.md

```diff
@@ -0,0 +1,150 @@
+     Free slots: 3/3
+    Advertising: no
+       Contexts:
+    [ 0] state: 0x8000 - unused
+    [ 1] state: 0x8000 - unused
+    [ 2] state: 0x8000 - unused
+
+Once this if working, you need to tell the node to accept incoming connections
+and start advertising itself:
+
+    ble adv
+
+Thats it, the node can now be seen and connected to from Linux.
+
+
+## Establishing a connection from Linux
+
+### Preparing Linux
+
+First of all, the Linux kernel needs to be configured to use 6lowpan over BLE.
+This needs to be done only once per session and consists of the following steps:
+
+    # Log in as a root user (IMPORTANT, as the following steps will not work
+    # using explicit `sudo` for each step...)
+    sudo su
+
+    # Mount debugfs file system, already the case on many distributions
+    mount -t debugfs none /sys/kernel/debug
+
+    # Load 6LoWPAN module
+    modprobe bluetooth_6lowpan
+
+    # Enable the bluetooth 6lowpan module.
+    echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
+
+Next, we should verify that our Bluetooth device is configured properly:
+
+    # Look for available HCI devices.
+    hciconfig
+
+This should show us some information about the available Blutooth devices. If no
+device is listed here, something is wrong...
+
+
+### Connecting to the RIOT node
+
+When connecting to the RIOT device, you need to know the BLE address of the
+device. There are two ways to find out:
+
+a) use `ble info` or `ifconfig` on the RIOT device directly and copy the address
+
+b) use `bluetoothctl` on Linux to scan for the device. Once `bluetoothctl` has
+   started, issue `scan on` to start scanning. The default name for the RIOT
+   device is set to `RIOT-GNRC`, so you should see it pop up.
+
+c) use `hcitool lescan` on the Linux host (older tool, predecessor of
+   `bluetoothctl`...)
+
+Once you have the address, you simply connect Linux to RIOT using the following
+command:
+
+    # Put your device address here...
+    # Note: the 2 after the address denotes a BLE public random address, default
+    #       used by `nimble_netif`
+    echo "connect 00:AA:BB:XX:YY:ZZ 2" > /sys/kernel/debug/bluetooth/6lowpan_control
+
+Thats it, you now have a IPv6 connection to your RIOT node. You can verify this
+using `ifconfig`, where something like the following should be visible:
+
+    bt0       Link encap:UNSPEC  HWaddr 00-19-86-00-16-CA-00-00-00-00-00-00-00-00-00-00
+          inet6 addr: fe80::19:86ff:fe00:16ca/64 Scope:Link
+          UP RUNNING MULTICAST  MTU:1280  Metric:1
+          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:18 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:1000
+          RX bytes:0 (0.0 B)  TX bytes:1624 (1.6 KB)
+
+You should also be able to ping your RIOT node. The devices link local address
+is also printed when running `ble info` on your RIOT device.
+
+    # Substitute the actual device address,
+    ping6 fe80::00aa:bbff:fexx:yyzz%bt0
```

Comment:
This might lead to confusion, as `0`, `a`, and `b` are valid hex characters, but `x`, `y`, and `z` are not. A less experienced user might expect only the 3 least significant bytes to be different.